setup.py

```diff
@@ -2,11 +2,22 @@
 import sys
 from setuptools import setup, find_packages, Command
 
+SEP='<><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>'
```

Comment:
SEP='<>'*40 ?
And also, this is only used inside Doctest class, please move to this class.